graphs/possible_bipartition.py

```diff
@@ -1,12 +1,38 @@
 # Can be used for BFS
-from collections import deque 
+
+from collections import deque
+
 
 def possible_bipartition(dislikes):
     """ Will return True or False if the given graph
         can be bipartitioned without neighboring nodes put
         into the same partition.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n+e)
+        Space Complexity: O(n)
```

Comment:
✨ Very nice BFS implementation. However, I believe you may not be accounting for an edge case. Consider this adjacency matrix:
`[
    [],
    [],
    [3,4],
    [2,4],
    [2,3]
  ]`
  
  Why might your code not account for this case?